conda/core/subdir_data.py

```diff
@@ -346,8 +346,12 @@ def _check_pickled_valid():
         return _pickled_state
 
     def _process_raw_repodata_str(self, raw_repodata_str):
-        json_obj = json.loads(raw_repodata_str or '{}')
-
+        try:
+            json_obj = json.loads(raw_repodata_str or '{}')
+        except json.decoder.JSONDecodeError:
+            log.debug("Conda repository may be experiencing issues, "
+                      + "please try again later or use another mirror")
+            raise
```

Comment:
We need to handle this decode error, but I don't think `log.debug()` is sufficient. Most users won't even see that. We need to throw the whole downloaded file away and start again.

Reply:
That's fair. I'm not familiar enough with conda's codebase to make that change; this is more of a bandaid to help those installing over CLI to see what's really going on and how to fix it (which is to wait, unless someone actually wants to implement the change you suggested)